etc/async-aurora/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "async-aurora"
+version = "0.1.0"
+authors = ["Aurora <hello@aurora.dev>"]
+edition = "2021"
+publish = false
+
```

Comment:
let's also add the description.

Reply:
Fixed at f004ca6d67fd0724e03495fb0386ebac11132ce5